src/modules/CMakeLists.txt

```diff
@@ -32,20 +34,23 @@ function(add_module_custom_target moduleDir targetName mimName recipeName)
             add_subdirectory(${_moduleDir})
         elseif(EXISTS ${CMAKE_CURRENT_SOURCE_DIR}/test/recipes/${recipeName}Tests.json)
             add_subdirectory(${_moduleDir})
-            add_dependencies(modulerecipeconfiguration ${_targetName})
+            if (BUILD_TESTS)

```

Comment:
> if (BUILD_TESTS) [](http://example.com/codeflow?start=11&length=17)

Thanks for adding this! Shouldn't this be even higher, for example to cover line 33 (JQ)?

Reply:
Actually, the entire block of testing modules, all from line 13 down to line 52

Reply:
I temporarily added it this way so that the current modulestest are not broken, all will change when JQ is removed as a part of the modulestest work.

Reply:
We still need that `add_subdirectory` line for each module, only the dependency and building of the test its self are isolated by the `BUILD_TESTS` flag.

Reply:
OK! Still JQ can be skipped unless test is enabled.